tutorial/numerical-entry.go

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
This file is `.go` it needs to be `.md` to work

Reply:
Wow. That is an embarrassing mistake. Sorry. Fixed now.